src/SAWScript/Crucible/LLVM/Builtins.hs

```diff
@@ -2016,16 +2018,35 @@ llvm_conditional_points_to_at_type ::
   AllLLVM SetupValue ->
   LLVMCrucibleSetupM ()
 llvm_conditional_points_to_at_type cond ptr ty val =
-  llvm_points_to_internal False (Just ty) (Just cond) ptr val
+  llvm_points_to_internal (CheckAgainstCastedType ty) (Just cond) ptr val
+
+-- | When invoking @llvm_points_to@ and friends, should SAW check that the type
+-- of the RHS value is compatible with a particular type?
+data CheckPointsToType
+  = Don'tCheckType
```

Comment:
Would it be too annoying to use `DoNotCheckType` here? (They're the same number of characters... hehehe)

Reply:
I'm not attached to the name `Don'tCheckType`, so I'd be fine with `DoNotCheckType`.

As a side note, I wonder if instead of defining a separate `CheckPointsToType` data type, I should move `CheckPointsToType` (currently in `saw-remote-api/src/Server.hs`) here and make the argument to `llvm_points_to_internal` be of type `Maybe (CheckPointsToType Type)` instead. This would avoid the need to explicitly convert between the two `CheckPointsToType` data types in `saw-remote-api/src/SAWServer/LLVMCrucibleSetup.hs`.